domain/storage/state/storagetypes.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+// These structs represent the persistent storage schema in the database.
+
+type ModelDetails struct {
```

Comment:
Do these types need to be exported and are we supposed to be prefixing with db or is that just some packages?

Reply:
Yeah, this was added here prematurely - deleting in this PR. Should have been in the next PR. Doesn't need to be exported. Some packages use db prefix. Many don't. We are inconsistent. The types already in storage package don't.